src/app/clusters/level-control/level-control.cpp

```diff
@@ -376,7 +376,7 @@ static bool shouldExecuteIfOff(EndpointId endpoint, CommandId commandId, uint8_t
         // ---------- The following order is important in decision making -------
         // -----------more readable ----------
         //
-        if (optionMask == 0xFF && optionOverride == 0xFF)
+        if (optionsMask == 0xFF && optionsOverride == 0xFF)
```

Comment:
As a followup: using `0xFF` here as a "no value" placeholder does not match the spec.  We should be using an `Optional` if there are callsites that don't have a value...  Please make sure that issue gets filed and labeled "spec".

Reply:
Agreed.  With the parameters always being pulled out from the arguments now, I think this may be dead code anyway, and likely the right  path is to remove this clause altogether.

Reply:
It's not dead code.  I thought it might be too, but I checked and there are things that do level operations in response to On/Off commands where these parameters really are not available.